docs/integrating-uptest-for-e2e-testing.md

```diff
@@ -0,0 +1,143 @@
+# Integrating Uptest for End to End Testing
+
+In this tutorial, we will integrate [uptest](https://github.com/upbound/uptest) to a Github repository to automate end to end
+testing managed resources. While we will use a _provider_ repository as an example, the process will be identical for a
+configuration repository.
```

Comment:
```suggestion
`Configurarion` repository.
```
Highlight Crossplane packages kinds to not confuse the reader with the arbitrary configuration repo